svn_select_to_git.py

```diff
@@ -892,6 +892,124 @@ def processRevision(exportDir, gitDir, log):
   # End if
 # End def processRevision
 
+############################################
+###
+### SVN Externals -> Externals_CAM functions
+###
+############################################
+
+def ReadExternalList(filename):
+    """Reads in text file list of CAM external directories/files"""
+
+    #Check if file exists:
+    if not os.path.exists(filename):
+        perr("External list, '{}', does not exist".format(filename))
+
+    ext_list = {}
+    try:
+        with open(filename) as f:
+            #Read in each line separately:
+            ext_list = f.readlines()
+     
+            #Remove any white space and endlines:
+            ext_list = [x.strip() for x in ext_list]                                
+
+    #Catch any errors, if present: 
+    except ValueError as e:
+        perr(e)
+    except Exception as e:
+        perr(e)
+
+    #Return external list:
+    return ext_list
+
+def git_external_svn_cam_remove(git_dir, ext_list, git_commit, git_com_msg):
+    """Removes all SVN external directories,
+       along with the 'SVN_EXTERNAL_DIRECTORIES' file,
+       in the cam sub-directory"""
+
+    #Determine current directory:
+    currdir = os.getcwd()
+
+    #Set CAM git directory:
+    cam_dir = git_dir+"/components/cam"
```

Comment:
Always use `os.path` functions (`os.path.join` in this case) when manipulating filenames.